jobs/BUILD

```diff
@@ -10,6 +10,13 @@ filegroup(
 )
 
 filegroup(
+    name = "scripts",
```

Comment:
rather than expose config_sort through the filegroup, you should export it as a `py_binary` or `py_library` which depends on the `ruamel_yaml` dependency you added, and then depend on that new rule in `prow/config/BUILD`.